packages/styles/scss/themes/expressive/components/_time-picker.scss

```diff
@@ -19,4 +19,9 @@
     height: carbon--rem(48px);
     width: carbon--rem(85px);
   }
+  .#{$prefix}--time-picker__select {
+    svg {
+      height: 100%;
```

Comment:
Not sure if setting the `height` changes anything

I think what @oliviaflory was trying to say in the ticket - if you compare the dropdown chevron and the time picker chevron, the time picker chevron looks bigger


<img width="387" alt="Screen Shot 2020-01-17 at 11 36 23 AM" src="https://user-images.githubusercontent.com/54281166/72629254-f413e200-391d-11ea-9a29-21d038a33ca4.png">
<img width="393" alt="Screen Shot 2020-01-17 at 11 36 39 AM" src="https://user-images.githubusercontent.com/54281166/72629256-f5dda580-391d-11ea-946a-7377793ae645.png">

Reply:
Okay got it

Reply:
@annawen1: after looking into this, the `Dropdown` component is using the lighter version of the chevron icon. And time picker is using a darker version icon. Where these icons are being referenced in the themes.